braindecode/samplers/base.py

```diff
@@ -65,6 +65,12 @@ def sample_recording(self):
         # XXX docstring missing
         return self.rng.choice(self.n_recordings)
 
+    def sample_classe(self):
```

Comment:
I don't understand this one, what does it do exactly? Why is it called sample_classe? anyways should be either class or classes